tests/system/Test/ControllerTesterTest.php

```diff
@@ -0,0 +1,203 @@
+<?php
+namespace CodeIgniter\Test;
+
+use CodeIgniter\Log\Logger;
+use Config\App;
+use Config\Services;
+use Tests\Support\Config\MockLogger as LoggerConfig;
+
+class ControllerTesterTest extends \CIUnitTestCase
+{
+
+	use ControllerTester;
+
+	public function setUp()
+	{
+		parent::setUp();
+	}
+
+	public function tearDown()
+	{
+		parent::tearDown();
+	}
+
+	public function testBadController()
+	{
+		$this->expectException(\InvalidArgumentException::class);
+		$logger = new Logger(new LoggerConfig());
```

Comment:
Wouldn't using this pollute the user's log files during testing? 

There's a MockLogger that can be used. I believe it's part of the CIUnitTestCase.

Reply:
Hehe - line 7 of that file ... use Tests\Support\Config\MockLogger as LoggerConfig;
The tests are already using the MockLogger :)

Reply:
Doh! Missed that. You were ahead of me. Yay!